interface/src/app/node/heavy_genes.tpl.html

```diff
@@ -0,0 +1,30 @@
+<div ng-if="!queryStatus">
+  <h4>
+    <div style="float: left">
+      <b>High-weight Genes: </b>{{genes.length}}
+    </div>
+    <div ng-if="genes.length > 0">
+      &nbsp({{hypoPercentage}}% Hypothetical)
```

Comment:
Browsers tend to be forgiving of typos. Most of the web would not work if they weren't! It is possible to run an HTML validator, too, which would point out such errors. GitHub highlighted the `&` in red because it was an error, in fact.